packages/cardano-services/src/util/schedule.ts

```diff
@@ -0,0 +1,29 @@
+import * as fs from 'fs';
+
+export type ScheduleConfig = {
+  queue: string;
+  cron: string;
+  data: object;
+  scheduleOptions: object;
+};
+type RawSchedulesConfig = {
+  schedules: {
+    sharedScheduleOptions: {};
+    list: Array<ScheduleConfig>;
+  };
+};
+export const readScheduleConfig = (filePath: string): Array<ScheduleConfig> => {
+  try {
+    const { schedules } = JSON.parse(fs.readFileSync(filePath, 'utf-8')) as RawSchedulesConfig;
```

Comment:
As per my experience, the use synchronous `fs` functions is fine in scripts (build scripts and so on..) while in the services I would use asynchronous version of the same functions.

Since applying the change I'm suggesting implies a `readScheduleConfig` signature change with consequential changes where it is called, this may require a considerable amount of time. Please raise this in next team meetings to check together if applying this change actually worth.

Reply:
I am not sure if I understood the suggestion's value. Let's talk at a team meeting and update back here.

Reply:
Being this called by argument parser (which requires synchronous functions) I'm fine keeping this as is